cpp/lib/include/AppArmorUtil.h

```diff
@@ -40,11 +40,11 @@ std::vector<std::pair<std::string, ProfileMode>> GetProfiles();
 /** \brief  Return the mode for the given profile. */
 ProfileMode GetProfileMode(const std::string &profile_id);
 
-/** \brief  Install the profile from the given path. Filename will later be used as profile_id. */
-void InstallProfile(const std::string &profile_path);
+/** \brief  Install the profile from the given path as local profile. Filename will later be used as profile_id. */
+void InstallLocalProfile(const std::string &profile_path);
```

Comment:
"as a local profile".  Should "Filename" be "profile_path"?

Reply:
Done... its just not marked as "outdated" because line+1 was marked with the comment.